cc-visionary/StickerMe
# [US 13 & 21] Admin Orders and Re-use of Contact Details

## Tasks Done:
- Show the orders in the admin orders page
![image](https://user-images.githubusercontent.com/22016684/132791062-66bef254-9fed-47ea-b1ec-4b3e1a52acac.png)
- Allow the customer to re-use an existing contact that was already saved
![image](https://user-images.githubusercontent.com/22016684/132791332-2323cd5b-1631-4202-981a-8bfa9f5ed56c.png)

## To Test the Admin Order Page (US21):
1. Login as username `customer` with password `password`
2. Add a new order by going to `Character Page`, inputting the necessary details including the Contact Details
2. Logout then login as username `moderator` with password `password`
3. Then go to the click the `orders` tab in the navbar
4. You'll then see the orders with their corresponding details such as total price, parts, etc.

## To Test the Re-use of an Existing Contact Details (US13):
1. Once you've completed testing US21, you should now have an existing contact detail.
2. Just add a new character again, then on the contacts part, you can now select an existing contact details if you had it saved.
